releasenotes/notes/releasenotes/notes/jmxfetch-0-38-0-fd6e1b74a5ca153c.yaml

```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    JMXFetch upgraded to `0.38.0 <https://github.com/DataDog/jmxfetch/releases/0.38.0>`_
```

Comment:
nit: I find it nice to update the release note that mentioned 0.37.0, instead of a new note, since no released agent version has 0.37.0 either yet